src/pages/decentralizing-xmtp.md

```diff
@@ -127,7 +131,45 @@ Any node or client can issue `MisbehaviorReports` to identify and prove non-comp
 
 XMTP's distributed architecture, use of smart contracts, and participation from multiple nodes and clients ensure that the network operates independently even if any single organization - including Ephemera - ceases to exist.
 
-**17. Does this mean there is a token I can buy?**
+**17. What metadata is visible on the network?**
+
+| Content | Is visible |
+| --- | --- |
+| When you joined the network | ✅ |
+| When you add a new wallet or device to your account | ✅ |
+| What time you have been invited to a conversation | ✅ |
+| How many devices and wallets you have added to your account | ✅ |
+| Who invited you to the conversation | ❌ |
+| What conversations you have been invited to | ❌* |
+| Who is in any group | ❌* |
+| When you sent a message in a conversation | ❌ |
+| The contents of the messages you send | ❌ |
+| How many messages you have sent | ❌ |
+| When you consent to join a conversation | ❌ |
+| The name and description of any group | ❌ |
+| The permissions and admin roles of any group | ❌ |
+
+**There is a risk that an attacker reading all messages on the network could draw statistical correlations between otherwise unrelated messages to reveal members that members were invited to a group at the same time, and which group ID they were added to, by looking at messages sent at similar times. We plan to address this risk by adding jitter to message timing before the release of XMTP mainnet.*
```

Comment:
Is there an extra word here? "reveal members that members were invited"

Reply:
Good catch. Meaning is different depending on "that members" or "members that". I'll commit the former, since the latter implies member identities are revealed.